bin/cylc-run

```diff
@@ -62,8 +62,8 @@ class start(scheduler):
             __doc__, jset=True,
             argdoc=[("REG", "Suite name"),
                     ("[START_POINT]",
-                    """Initial cycle point or 'now';
-                               overrides the suite definition.""")])
+                     """Initial cycle point or 'now'; """
+                     """overrides the suite definition.""")])
```

Comment:
ditto.  The original has a second line with correct indent; your version prints one line 90+ chars long.